pkg/rpm/horizon-cli/horizon-cli.aarch64.spec

```diff
@@ -0,0 +1,76 @@
+# Spec file for the horizon-cli RPM. For info about writing spec files, see:
```

Comment:
Instead of duplicating the spec files for different architectures, why not just use the `%{_arch}` variable? I would prefer that.

Reply:
done.